code/__HELPERS/mobs.dm

```diff
@@ -67,6 +67,9 @@ proc/random_skin_tone(species = "Human")
 	else if(species == "Vox")
 		. = rand(1,6)
 		return .
+	else if(species == "Grey")
```

Comment:
this entire proc is an absolute nightmare, this is the exact reason for object oriented programming, so we don't need this

this proc should be on species datums and just be the segment for whatever species it is, e.g.:
```
/datum/species/grey/random_skin_tone()
   return rand(1,4)
```